service/session_http.go

```diff
@@ -0,0 +1,160 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	ae "joshsoftware/go-e-commerce/apperrors"
+	"joshsoftware/go-e-commerce/config"
+	"joshsoftware/go-e-commerce/db"
+	"net/http"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	logger "github.com/sirupsen/logrus"
+)
+
+//AuthBody stores responce body for login
+type AuthBody struct {
+	Message string `json:"meassage"`
+	Token   string `json:"token"`
+}
+
+//generateJWT function generates and return a new JWT token
+func generateJwt(userID int) (tokenString string, err error) {
```

Comment:
create `token` package & add `NewJWT(userID int) (tokenString string, err error)` func &
move all 
From: `generateJwt(userID int) (tokenString string, err error)`
To: `NewJWT(userID int) (tokenString string, err error)`